tests/ValueParsers/IsoTimestampParserTest.php

```diff
@@ -181,13 +198,13 @@ public function validInputProvider() {
 			'-1-01-04T00:00:00Z' => array(
 				'-0000000000000001-01-04T00:00:00Z',
 				TimeValue::PRECISION_10a,
-				$gregorian,
+				$julian,
```

Comment:
And I see no reason to mark them as Gregorian. That just makes the heuristic more complex, more surprising, and less consistent with existing conventions.